apps/mobile/src/screens/CreateGroup/CreateGroupVisibilityAccessibility.js

```diff
@@ -53,6 +57,26 @@ export default function CreateGroupVisibilityAccessibility ({ navigation }) {
             />
           ))}
         </View>
+
+        {visibility.value === GROUP_VISIBILITY.Public && (
+          <View className='mb-6 bg-foreground/5 p-4 rounded-lg'>
+            <Text className='text-foreground font-bold mb-2'>
+              {t('Optional') + ': ' + t('Allow my Group into the Commons')}
```

Comment:
Allow my group doesn't quite sound right - "Add my group to the commons?" might require a little discussions